convert2rhel/utils.py

```diff
@@ -643,3 +644,18 @@ def flatten(dictionary, parent_key=False, separator="."):
         else:
             items.append((new_key, value))
     return dict(items)
+
+
+def write_json_object_to_file(path, data, mode="w"):
```

Comment:
Oh, I thought mode would be the permissions.  So, `0o600`, for instance.

Reply:
done.